src/process.cpp

```diff
@@ -585,6 +585,16 @@ std::shared_ptr<DarlingServer::Thread> DarlingServer::Process::_pickS2CThread(vo
 	return thread;
 };
 
+const std::string& DarlingServer::Process::executablePath() const {
```

Comment:
Following the signature change from the header:
```suggestion
std::string DarlingServer::Process::executablePath() const {
```